.gitignore

```
@@ -23,3 +23,5 @@ coverage/*
 # Ignore rvm and rbenv files
 .ruby-version
 .ruby-gemset
+
+.yardoc/
```

To prevent the docs generated by `yard` from being committed :)

```suggestion
.yardoc/
doc/
```

---

Oh I just realized the `doc/` directory is already used. We should probably set a custom output directory in the yard config.